Makefile

```diff
@@ -11,6 +11,11 @@ generate: sync
 	./hack/bootstrap-ginkgo.sh
 	(cd tools/openapispec/ && go build)
 	tools/openapispec/openapispec --dump-api-spec-path api/openapi-spec/swagger.json
+	(chmod -R 777 hack/gen-swagger-doc && docker run --rm \
```

Comment:
@rmohr this `chmod` finally helped me to overcome issue with permissions during gradle execution inside of container.
but it has nasty side effect, that it change mode of files in repository :-(
```diff
diff --git a/hack/gen-swagger-doc/build.gradle b/hack/gen-swagger-doc/build.gradle
old mode 100644
new mode 100755
diff --git a/hack/gen-swagger-doc/gradle.properties b/hack/gen-swagger-doc/gradle.properties
old mode 100644
new mode 100755
```

Reply:
Maybe run the script with sudo and the uid of the user which owns the mounted directory?

Reply:
`docker run ... sudo -u X bash hack/gen-swagger-doc/gen-swagger-docs.sh`
apparently `sudo` is not inside of gradle container ... 
but I tried to set user for docker command `docker run -u X ...` and it works for me ... I will try that